### PR TITLE
refactors pods, new helper for spawning items from pods through code

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Admins have some ease of use in podspawning for IC deliveries, but as it is it's kind of a dumb copypastapalooza for coders who want to do the same.

## Why It's Good For The Game

Easier to add IC deliveries, to show you the power of podspawn_path i refactored all but all instance of creating a pod via code into this

## Changelog
:cl: 
refactor: added podspawning helper codewise, most code pod spawns use it
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
requested by @Mothblocks

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by tralezab** [comment]
Sorry to my middle eastern/northern african fans

**Comment by maxymax13** [comment]
a pr with unicode garbage in its name should automatically deduct 20 gbp

**Comment by SuperNovaa41** [comment]
arm has pushed his pr names too far

**Comment by MrDoomBringer** [comment]
Good stuff, nice and simple podspawning 👍 
I should really refactor supplyopds

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1474,3 +1474,49 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
...
+ * Arguments:
+ * * specifications: special mods to the pod, see non var edit specifications for details on what you should fill this with
+ * Non var edit specifications:
+ * * target = where you want the pod to drop
+ * * path = a special specific pod path if you want, this can save you a lot of var edits
+ * * style = style of the pod, defaults to the normal pod
+ * * spawn = spawned path or a list of the paths spawned, what you're sending basically
+ * Returns the pod spawned, in case you want to spawn items yourself and modify them before putting them in.
+ */
+/proc/podspawn_path(specifications)
```
Comment (Mothblocks) [inline code/__HELPERS/unsorted.dm:1491]: Does the name `podspawn_path` make sense anymore with the new signature? Why not just `podspawn`?

Comment (tralezab) [reply code/__HELPERS/unsorted.dm:1491]: i wanted to do podspawn_path, but uuh... podspawn is the admin verb! I mean, maybe i'll still do it

Comment (Mothblocks) [reply code/__HELPERS/unsorted.dm:1491]: `podspawn` doesn't look like it's the name of the verb, though.

Comment (Mothblocks) [reply code/__HELPERS/unsorted.dm:1491]: As in, the name of the proc of the verb.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Name it podspawn.

**Pull request merged**